clrfeatures.cmake

```diff
@@ -14,7 +13,9 @@ if(NOT DEFINED FEATURE_EVENT_TRACE)
   endif()
 
   if(CLR_CMAKE_PLATFORM_LINUX)
-    if(CLR_CMAKE_TARGET_ARCH_AMD64)
+    if(CLR_CMAKE_TARGET_TIZEN_LINUX)
+      set(FEATURE_EVENT_TRACE 1)
```

Comment:
As I commented at https://github.com/dotnet/coreclr/pull/14137#discussion_r140676657, we can get build result as you want if we only remove `set(FEATURE_EVENT_TRACE 1)` at line 2.  Is this condition necessary?

Reply:
I'm not sure, but this feature may be required for x86/Linux. So, I would like to introduce this line for consistency.

Reply:
@parjong I agree with you that making it the same on all platforms is good.